docs/components/block-quote/README.md

```diff
@@ -172,8 +172,8 @@ This component has compliance with WCAG guidelines by:
 </cdr-doc-table-of-contents-shell>
 </template>
 
-<template slot="Design Guidelines">
-  <cdr-doc-table-of-contents-shell tab-name="Design Guidelines">
+<template slot="Guidelines">
+  <cdr-doc-table-of-contents-shell tab-name="Guidelines">
```

Comment:
I updated `tab-name` here, but I don't think we really need this prop because most of the others don't have it.

Reply:
that prop is necessary for this component, otherwise it won't show the correct active tab